operations/transaction-service/src/common/jwt-clientid.middleware.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com). All Rights Reserved.
+//
+// This software is the property of WSO2 LLC. and its suppliers, if any.
+// Dissemination of any information or reproduction of any material contained
+// herein in any form is strictly forbidden, unless permitted by WSO2 expressly.
+// You may not alter or remove any copyright or other notice from copies of this content.
+
+import { Injectable, NestMiddleware, UnauthorizedException } from '@nestjs/common';
+import { Request, Response, NextFunction } from 'express';
+import { decode } from 'jsonwebtoken';
+
+@Injectable()
+export class JwtClientIdMiddleware implements NestMiddleware {
+  use(req: Request, res: Response, next: NextFunction) {
+    const jwtHeader = req.headers['x-jwt-assertion'] || req.headers['X-JWT-Assertion'];
```

Comment:
```suggestion
    const jwtHeader = req.headers['x-jwt-assertion'];
```
Header names are case insensitive. I don't think we need this OR operation.